build/images/base/Dockerfile.ubuntu

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04 as cni-binaries
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates
+
+# Leading dot is required for the tar command below
+ENV CNI_PLUGINS="./host-local ./loopback ./portmap"
+
+RUN mkdir -p /opt/cni/bin && \
+    wget -q -O - https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
+
+
+ARG OVS_VERSION=2.13.0
```

Comment:
needs to be at the top of the Dockerfile or it won't work, I just relalized

Reply:
Thanks, updated.